src/widgetastic_patternfly/__init__.py

```diff
@@ -843,9 +844,12 @@ class BootstrapTreeview(Widget):
     You don't have to specify the ``tree_id`` if the hosting object implements ``tree_id``.
 
     Args:
-        tree_id: Id of the tree, the closest div to the root ``ul`` element.
+        tree_id: Id of the tree, the closest div or ``miq-tree-view`` to the root ``ul`` element.
     """
-    ROOT = ParametrizedLocator('#{@tree_id}')
+    ROOT = ParametrizedLocator('|'.join([
+        './/miq-tree-view[@name={@tree_id}]/div',
```

Comment:
I believe this `tree_id` needs to be quoted here.

Failing for me in testing on miq build from master.20170918, and searching in the xpath this ROOT lookup fails without the value being quoted.